tests/generate/test_generate_generators.test_generate_generators_valid_generator_no_description.approved.txt

```diff
@@ -0,0 +1,9 @@
+Usage: algokit generate [OPTIONS] COMMAND [ARGS]...
```

Comment:
This looks the same as the others, do we expect a difference?

Reply:
@robdmoore this was also an artifact from the test file before it was renamed, see all tests under test_generate_custom_generate_commands.* prefix